Anytype/Sources/PresentationLayer/Common/SwiftUI/EqualFitWidthHStack.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import SwiftUI
+
+struct EqualFitWidthHStack: Layout {
```

Comment:
**EqualWidthHStack** looks like shorter name with the same meaning

Reply:
Fill variant can be implemented without custom layout. I wanted to show that this just only for fit mode.